core/sql/executor/HBaseClient.java

```diff
@@ -587,13 +588,11 @@ public boolean alter(String tblName, Object[] tableOptions, long transID)
         ChangeFlags status = 
             setDescriptors(tableOptions,htblDesc /*out*/,colDesc /*out*/, defaultVersionsValue);
 
-        // TODO: Remove this stub once transactional alter method is added to table member
-        transID = 0;  // stub
-
         try {
             if (transID != 0) {
-                // TODO: write this method to support transactional alter
-                // table.alter(tblName, tableOptions, transID);
```

Comment:
There are some commented code here. I guess this is to integrate the remaining code later.  Other than that, everything else looks good.